lib/rollout.rb

```diff
@@ -21,6 +23,7 @@ def initialize(name, string = nil, opts = {})
         @percentage = raw_percentage.to_f
         @users = users_from_string(raw_users)
         @groups = groups_from_string(raw_groups)
+        # @updated_at = Time.now.strftime(Rollout::TIME_FORMAT)
```

Comment:
this is because I was thinking on add a `@updated_at` variable 🤔 but I didn't follow with this approach